### ISSUE TITLE
Sway reload ignores bindswitch

### ISSUE THREAD

**Issue by lovesegfault** [swaywm/sway#5579]
- **Sway Version:**
```
viins ❯ swaymsg -t get_version
sway version 1.5
```

- **Debug Log:**
https://gist.github.com/4655f0384d87cb922d04da624077e761

- **Configuration File:**
https://gist.github.com/1e4a52fbef3686f1bb277199800c3f26

- **Description:**
In my sway config I have the following:
```
bindswitch --locked lid:on output eDP-1 disable
bindswitch --locked lid:off output eDP-1 enable
```
in order to disable my laptop screen while I have it in clamshell mode, which is how I run it almost always. This works perfectly, except that I've noticed that if I reload sway it will re-enable that output and I have to manually disable it with `swaymsg`.

In other words, reloading sway will enable outputs that should be disabled by the bindswitch.

**Comment by Emantor** [comment]
Add `--reload` to the bindswitch command:
```
If the --reload flag is given, the binding will also be executed when the config is reloaded.
toggle bindings will not be executed on reload.
The --locked flag will operate as normal so if the config is reloaded while locked and
--locked is not given, the binding will not be executed.
```
from `man 5 sway`.

**Comment by lovesegfault** [comment]
Ah, yes, that works!

Regardless, this seems a bit un-intuitive, I don't need to pass `--reload` to other config commands. Maybe someone could help explain the rationale for the default behavior to be as it is?

**Comment by RedSoxFan** [comment]
> Regardless, this seems a bit un-intuitive, I don't need to pass --reload to other config commands. Maybe someone could help explain the rationale for the default behavior to be as it is?

`bindswitch` is a binding that gets triggered when a switch event occurs, just like with `bindsym/bindcode` for keyboard and mouse. No switch event is emitted when the config is reloaded - as it does not change state - so the binding does not get triggered. The `--reload` flag was added to allow for an artificial event to be emitted on reload to trigger the binding.
